lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -314,18 +285,19 @@ class StandardAccountTest {
         // Compiled contract format sometimes changes, this causes changes in the class hash.
         // If this test starts randomly falling, try recalculating class hash.
```

Comment:
This warning applies only to Cairo 0 contracts, as we do not need to manually compute class hash for Cairo 1 contracts, and therefore can be removed.

Reply:
Removed.